src/ia-bookreader/ia-bookreader.js

```diff
@@ -105,7 +105,7 @@ export class IaBookReader extends LitElement {
 
   render() {
     return html`
-      <div class="ia-bookreader">
+      <div class="main-component">
```

Comment:
Gaah, https://github.com/webcomponents/polyfills/issues/477 ! Let's give this a tighter class name to avoid hitting other things; I started doing `*_root`; e.g. `ia-bookreader__root` so that it can't possibly in any way break 😅

Reply:
TIL about polyfills class naming